packages/ui-components/src/components/ai-chat-messages/types.ts

```diff
@@ -1,21 +1,28 @@
 import { SourceCode } from '@openops/shared';
 
+/**
+ * @deprecated use shared types instead from ai sdk
+ */
 export enum AIChatMessageRole {
   user = 'user',
   assistant = 'assistant',
 }
 
+/**
+ * @deprecated use shared types instead from ai sdk
+ */
 export type AIChatMessageContent =
   | string
   | {
-      type: 'structured';
       parts: (
         | { type: 'text'; content: string }
-        | { type: 'code'; content: string; language?: string }
         | { type: 'sourcecode'; content: SourceCode }
       )[];
     };
 
+/**
+ * @deprecated use shared types instead from ai sdk
+ */
```

Comment:
deprecated those as previously discussed, we need to use structure from ai SDK for multi-part messages